unitary/examples/quantum_rpg/input_helpers.py

```diff
@@ -54,9 +55,32 @@ def get_user_input_number(
             print("number out of range", file=file)
 
 
+def get_multiple_user_inputs(
+    get_user_input: _USER_INPUT,
+    *prompts: Sequence[Union[Callable[[], int], Callable[[], str]]],
+    file: TextIO = sys.stdout,
+) -> Sequence[Union[int, str]]:
+    """Runs multiple number or string prompts and returns their results.
+
+    After all inputs have been provided the last prompt asks for confirmation if
+    user happy with inputs, and allows to restart the sequence and change
+    inputs.
+    """
+    while True:
+        inputs = [p() for p in prompts]
+        print("[enter]) Confirm selection.", file=file)
```

Comment:
I am conflicted about making the user do an extra prompt at the end for every action.  Let's discuss at the next unitary meeting.